src/helpers/create-pr.ts

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2021 Expedia, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { context } from '@actions/github';
+import { octokit } from '../octokit';
+
+interface CreatePR {
+  title: string;
+  body: string;
+}
+
+export const createPr = ({ title, body }: CreatePR) => {
+  octokit.pulls.create({
+    title,
+    head: context.ref.replace('refs/heads/', ''),
+    base: 'HEAD',
+    body,
+    maintainer_can_modify: true,
+    draft: false,
+    issue: undefined,
```

Comment:
Nit pick, but we can omit these since they are not required

Reply:
Done.